changelog.md

```diff
@@ -3,4 +3,4 @@
 * [BUG] Fixed Modrinth Uploads failing when using ModLoaders enum - HypherionSA
 * [CHANGE] Deprecated `version` property, as it clashes with Gradle's version property. Replace with `projectVersion` - HypherionSA
 * [FEAT] Support Gradle JavaVersions in CurseForge setJavaVersion - [#9](https://github.com/firstdarkdev/modpublisher/pull/9) - MattSturgeon
-* [FEAT] Experimental GitHub release mutations - [#8](https://github.com/firstdarkdev/modpublisher/pull/8) - MattSturgeon
\ No newline at end of file
+* [FEAT] Experimental GitHub release options - [#8](https://github.com/firstdarkdev/modpublisher/pull/8), [#11](https://github.com/firstdarkdev/modpublisher/pull/11) - MattSturgeon
```

Comment:
Q: Are both these changes in the same release?

Reply:
PR 8 was released with 2.1.0, so this one will be with 2.1.1. Actually shouldn't have released 2.1 yet, but had to sneak in a bug fix :facepalm:

Reply:
In that case are you happy with this changelog or would you rather it be split?

Reply:
Done